Library/Formula/kibana.rb

```diff
@@ -0,0 +1,63 @@
+class Kibana < Formula
+  homepage "https://www.elastic.co/products/kibana"
+  url "https://download.elastic.co/kibana/kibana/kibana-4.0.2-darwin-x64.tar.gz"
+  version "4.0.2"
+  sha256 "3792e423dd97a216e12b0bb1ee97cf75712b2f850422d3746ff076ce9e1f2e01"
+  head "https://github.com/elastic/kibana.git"
+
+  def install
+    prefix.install %w[ node src ]
+
+    etc_path.mkpath
+
+    var_path.mkpath
+
+    conf_path = etc_path/"kibana.yml"
+
+    cp "config/kibana.yml", conf_path unless conf_path.exist?
+    (prefix/"config").install_symlink conf_path
+
+    (var_path/"plugins").mkpath
+    prefix.install_symlink var_path/"plugins"
+
+    bin.install "bin/kibana"
+  end
+
+  def caveats; <<-EOS.undent
+    Modify #{etc_path}/kibana.yml to set the elasticsearch_url to your Elasticsearch instance
+
+    Plugins are installed to #{var_path}/plugins
+  EOS
+  end
+
+  plist_options :manual => "kibana"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN"
+    "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>Program</key>
+        <string>#{opt_bin}/kibana</string>
+        <key>RunAtLoad</key>
+        <true/>
+      </dict>
+    </plist>
+  EOS
+  end
+
+  def etc_path
+    etc/"kibana"
+  end
+
+  def var_path
+    var/"lib/kibana"
+  end
+
+  test do
+    assert_equal "4.0.2\n", `kibana -V`
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!